web/src/components/tasks/business-formation/review/ReviewNonprofitProvisions.tsx

```diff
@@ -0,0 +1,160 @@
+import { Content } from "@/components/Content";
+import { ExpandCollapseString } from "@/components/ExpandCollapseString";
+import { ReviewSubSection } from "@/components/tasks/business-formation/review/section/ReviewSubSection";
+import { BusinessFormationContext } from "@/contexts/businessFormationContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { ReactElement, useContext } from "react";
+
+export const ReviewNonprofitProvisions = (): ReactElement => {
+  const { state } = useContext(BusinessFormationContext);
+  const { Config } = useConfig();
+
+  const fillInByLawFormText = (stringToCompare: string | undefined, outputText: string): string => {
+    if (stringToCompare === "IN_BYLAWS") {
+      return `${outputText} ${Config.formation.fields.isVeteranNonprofit.inByLaws}`;
+    }
+
+    if (stringToCompare === "IN_FORM") {
+      return `${outputText} ${Config.formation.fields.isVeteranNonprofit.inForm}`;
+    }
+
+    return `${outputText}:`;
+  };
+
+  const isVisibleInReview = (stringToCompare: string | undefined): boolean => {
+    if (stringToCompare === "IN_FORM") {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const showQuestionAnswer = (questionString: string | undefined, outputText: string): ReactElement => {
+    return (
+      <>
+        <Content style={{ display: "inline-block" }}>
+          {fillInByLawFormText(questionString, outputText)}
+        </Content>
+        {questionString === undefined && (
+          <div style={{ display: "inline-block" }}>
+            &nbsp;
+            <span className={"bg-accent-warm-extra-light text-italic"}>
+              {Config.formation.general.notEntered}
+            </span>
```

Comment:
[pebble] Can we use the `<ReviewNotEntered />` component here?

Reply:
done